src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -551,30 +551,25 @@ private static ImmutableArray<ISymbol> GetMembers(INamedTypeSymbol type, ISymbol
 
         private static bool IsOverridable(ISymbol member, INamedTypeSymbol containingType)
         {
-            if (member.IsAbstract || member.IsVirtual || member.IsOverride)
-            {
-                if (member.IsSealed)
-                {
-                    return false;
-                }
+            if (!member.IsAbstract && !member.IsVirtual && !member.IsOverride)
+                return false;
 
-                if (!member.IsAccessibleWithin(containingType))
-                {
-                    return false;
-                }
+            if (member.IsSealed)
+                return false;
 
-                switch (member.Kind)
-                {
-                    case SymbolKind.Event:
-                        return true;
-                    case SymbolKind.Method:
-                        return ((IMethodSymbol)member).MethodKind == MethodKind.Ordinary;
-                    case SymbolKind.Property:
-                        return !((IPropertySymbol)member).IsWithEvents;
-                }
-            }
+            if (!member.CanBeReferencedByName)
+                return false;
```

Comment:
this is the new check.  the rest is just making teh method consistent with flow control.